src/datreant/core/util.py

```diff
@@ -17,3 +20,35 @@ def makedirs(path):
             pass
         else:
             raise
+
+
+def touch_me(path):
+    Path(path).touch()
+
+
+def fullpath(path):
+    if not path:
+        return None
+    if not isinstance(path, six.string_types):
+        path = str(path)
+    path = os.path.expanduser(path)
+    if not os.path.isabs(path):
+        path = os.path.abspath(path)
+    return path
+
+
+def isfullpath(path):
+    return path == fullpath(path)
+
+
+def path_leaf(path):
+    return os.path.basename(os.path.normpath(path))
+
+
+def relpath(path):
+    """Returns *path* on Windows, and relative path elsewhere."""
```

Comment:
Does a Windows filesystem have no concept of a relative path? Unfortunately I have very little experience with trying to do anything Python on Windows, so apologies for the ignorance on my part.